README.md

```diff
@@ -227,4 +227,14 @@ beforeEach(
     fixture.detectChanges();
   })
 );
+```
+
+## Using scripts/resources.js file
+
+This file is used to convert the source App Icon and Splash screen image to the target App Icon and Splash screen image size for both IOS and Android mobile's for any responsive and version's. It contains all size target and source images name in it. The command is added in the package.json file in Script tag.
```

Comment:
Rather than just saying "there is a command," it's generally nice to provide the actual command. Otherwise the user has to go looking for it in package.json. You can see examples of this elsewhere in this readme.

Additionally, I tried to run this command and got the following error:

```
❯ npm run resources  

> rosie-prototype@0.0.1 resources
> cordova-res ios && cordova-res android && node scripts/resources.js

sh: cordova-res: command not found
```

Are there prerequisites for running this command? When would I run it? Does this have anything to do with building the native binaries? How do I do that?

Reply:
You Need to install the cordova-res plugin 
npm i cordova-res

Reply:
Should that be added to the devDependencies?

Reply:
added in devDependencies file

Reply:
Thanks. I see you have both `cordova-res` added to the devDependencies (good), and also instructions to install `cordova-res` manually (which is redundant). Can you remove the explicit step to install it manually?

Also, I get the following when I run this locally:

```bash
❯ npm run resources

> rosie-prototype@0.0.1 resources
> cordova-res ios && cordova-res android && node scripts/resources.js

WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
Generated 47 resources for iOS
Wrote to config.xml
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
WARN:	Must perform conversion from jpeg to png.
Generated 18 resources for Android
Wrote to config.xml
~/MVP-App-TestHarness/scripts/resources.js:222
      if (err) throw err;
               ^

[Error: ENOENT: no such file or directory, copyfile 'resources/ios/icon/icon-20.png' -> 'ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-20x20@1x.png'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'copyfile',
  path: 'resources/ios/icon/icon-20.png',
  dest: 'ios/App/App/Assets.xcassets/AppIcon.appiconset/AppIcon-20x20@1x.png'
}
```

I think this relates to my previous comment about what this script does and how it fits into the build process:

> Are there prerequisites for running this command? When would I run it? Does this have anything to do with building the native binaries? How do I do that?

Can you help me understand?

Reply:
This is how it runs it convert the icon and splash for the source targets, it just it's running process and that error for resources/ios/icon/icon-20.png is may be it already converted so it's occur and we will check it,